homeassistant/components/decora/light.py

```diff
@@ -61,6 +79,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         device["name"] = device_config[CONF_NAME]
         device["key"] = device_config[CONF_API_KEY]
         device["address"] = address
+        _LOGGER.warning(device["name"])
```

Comment:
We can remove this, right?

Reply:
Yes definitely, removed in https://github.com/home-assistant/home-assistant/pull/28279/commits/4dac56d76575351e124d2918bc9c73df690d0fc4